secret-detector/src/test/java/com/expedia/www/haystack/pipes/secretDetector/SpringConfigTest.java

```diff
@@ -25,6 +25,7 @@
 import com.expedia.www.haystack.pipes.commons.TimersAndCounters;
 import com.expedia.www.haystack.pipes.commons.health.HealthController;
 import com.expedia.www.haystack.pipes.commons.health.HealthStatusListener;
+import com.expedia.www.haystack.pipes.commons.kafka.KafkaConfigurationProvider;
```

Comment:
Can we revert this class?

Reply:
To support the new way of configuration and also give backward compatibility, we need to add this